Sources/SwiftGen/ParserCLI.swift

```diff
@@ -55,7 +55,7 @@ let allParserCommands: [ParserCLI] = [
     pathDescription: "Directory to scan for .storyboard files. Can also be a path to a single .storyboard"
   ),
   .init(
-    parserType: Yaml.Parser.self,
+    parserType: JSON.Parser.self,
```

Comment:
Was there some legacy error there? (I honestly couldn't remember if we have two really distinct parsers for JSON vs Yaml now or only one for both)

Reply:
Nah, it used to be the same parser.

Now, because they define different default filters, the JSON one is a subclass of the YAML parser that only overrides that default filter.

Reply:
Haw, which explains why it's not `final` anymore (comment below). Missed that. :+1: